app/src/main/java/com/example/timemanagementapp/DTO/ActivityDTO.kt

```diff
@@ -1,7 +1,7 @@
 package com.example.timemanagementapp.DTO
 
 class ActivityDTO {
-    var activityName: String? = null
-    var activityDuration = 0
+    private var activityName: String? = null
```

Comment:
Made updates to visibility modifiers.

Reply:
Yes... good idea, and this aligns with best practices.